hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/domain/EntityIdServiceImplTest.java

```diff
@@ -260,6 +260,15 @@ void storeNull() {
         assertDoesNotThrow(() -> entityIdService.notify(null));
     }
 
+    @Test
+    void lookupCreate2EvmAddress() {
```

Comment:
Perhaps something more specific to the fact that the account has the EVM address

```suggestion
    void lookupAccountWithEvmAddress() {
```

Reply:
Sounds good, I have renamed the method.